docs/guides/faq.md

```diff
@@ -74,6 +74,13 @@ help but are outside of the scope of video.js.
 
 For content that must be highly secure [videojs-contrib-eme][eme] adds DRM support.
 
+## Q: Can I turn off video.js logging?
+Yes! This can be achieved by adding the following code _before_ including video.js:
+
+```js
+window.VIDEOJS_DEFAULT_LOG_LEVEL = 'off';
```

Comment:
I think the only way to do this would be to really refactor the logging to create new console wrappers for each player (alongside the one for video.js itself). Probably not worth doing for a minor.